app/test/search/handlers_test_utils.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:shelf/shelf.dart' as shelf;
+
+import 'package:pub_dartlang_org/search/handlers.dart';
+
+Future<shelf.Response> issueGet(String path) async {
+  final uri = 'https://pub.dartlang.org$path';
```

Comment:
Shouldn't the URL be `search-dot-dartlang.org` (not that it matters for the test, but a reader might get confused)